src/pages/assets/AssetsMobilePage.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react'
 import cx from 'classnames'
 import { AutoSizer, List } from 'react-virtualized'
-import { Button, Label } from '@santiment-network/ui'
+import { Label } from '@santiment-network/ui'
```

Comment:
`[D.1]` Better to use an default import: `import Label from '@santiment-network/ui/Label'` (Otherwise webpack includes whole library)